dbUtils.js

```diff
@@ -11,11 +11,11 @@ const getPolls = async () => {
 
 const createPoll = async (request) => {
     const {
-        pollTitle, choices, isClosed, channelName, channelId, teamName, teamId,
+        pollTitle, choices, isClosed, channelName, channelId, teamName, teamId, isGame
     } = request;
-    return db.query('INSERT INTO public.polls (poll_title, choices, is_closed, channel_name, channel_id, team_name, team_id)' +
-        'VALUES($1, $2, $3, $4, $5, $6, $7)',
-        [pollTitle, choices, isClosed, channelName, channelId, teamName, teamId]).then((results) => {
+    return db.query('INSERT INTO public.polls (poll_title, choices, is_closed, channel_name, channel_id, team_name, team_id, is_game)' +
+        'VALUES($1, $2, $3, $4, $5, $6, $7, $8)',
+        [pollTitle, choices, isClosed, channelName, channelId, teamName, teamId, isGame]).then((results) => {
             return results;
```

Comment:
Do we want to store this is_game field in this table? Should we instead game a Game table and associate each poll with a game via a database relation ship, Then 'is_automod' could be a boolean in that table instead.
Maybe I'm misunderstanding what isGame is supposed to represent, could you clarify for me?

Reply:
its a game start poll, i did it this way because i had envisioned being able to do custom ones, but we can change the db design